packages/react-ui-components/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -38,12 +41,25 @@ export const KvCodeEditor = ({
 		editorRef.current.updateOptions(editorOptions);
 	}, [editorRef.current, readOnlyRef.current, editorOptions]);
 
+	const loadFont = useCallback(async () => {
+		await document.fonts.load(getFontName(initialOptions));
```

Comment:
What if the font cannot be loaded? Does this throw an error? Should we handle it?

Reply:
I'll add error handling for when the font fails loading, I believe this promise is rejected if that is the case, so a `try/catch` block should work here and would allow me to send some information to the implementing party of what happened. Thanks!

Reply:
Done @djcouto 
![image](https://user-images.githubusercontent.com/13225555/222414690-027c4215-16d5-4ecf-8650-f56163cc5419.png)

Reply:
Done @djcouto :) thanks for the heads up.
![image](https://user-images.githubusercontent.com/13225555/222414938-95addbb8-d0ea-4716-80b2-1d5f2194569f.png)